packages/forms-web-app/src/routes/examination/about-the-application.js

```diff
@@ -3,8 +3,8 @@ const router = express.Router();
 
 const aboutTheApplicationController = require('../../controllers/examination/about-the-application');
 
-router.get('/:page', aboutTheApplicationController.getAboutTheApplication);
-router.post('/search/:page', aboutTheApplicationController.postSearchDocument);
+router.get('/:case_ref/:page', aboutTheApplicationController.getAboutTheApplication);
+router.post('/:case_ref/search/:page', aboutTheApplicationController.postSearchDocument);
```

Comment:
@pk2453 Surely these routes need to be registered within `packages/forms-web-app/src/routes/examination/index.js` so that they don't contain `about-the-application` ?

Reply:
Well, its just one of way to write for easy navigation otherwise you can have all your routes in single file